pkgs/development/libraries/gdk-pixbuf/default.nix

```diff
@@ -31,11 +31,11 @@ in stdenv.mkDerivation rec {
   ]
     ++ stdenv.lib.optional stdenv.isDarwin fixDarwinDylibNames;
 
-  propagatedBuildInputs = [ glib libtiff libjpeg libpng jasper ];
+  propagatedBuildInputs = [ glib libtiff libjpeg libpng ];
 
   mesonFlags = [
     "-Ddocs=true"
-    "-Djasper=true"
+    "-Djasper=false"
```

Comment:
Jasper is off by default so you can just drop this.

Reply:
Yes, I wrote it that way after reading Gentoo's ebuild but you are right. I will correct it later today.

Reply:
Done